extensions/interactions/FractionInput/directives/FractionInputValidationService.js

```diff
@@ -124,6 +154,16 @@ oppia.factory('FractionInputValidationService', [
                 }
                 break;
               case 'HasIntegerPartEqualTo':
+                if (!allowNonzeroIntegerPart) {
```

Comment:
Strictly speaking, they might say "has integer part equal to 0", in which case what you have isn't quite right...

Reply:
Hmmm. Didn't think of that.. Will make that change. So while testing I test both situations right?

Reply:
Done